docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[indicators-of-compromise]]
+= Indicators of compromise
+
+The Intelligence page collects data from enabled threat intelligence feeds and provides a centralized view of indicators. This guide defines threat intelligence and indicators, helps you get started using the Intelligence page, and explains how to work with indicators.
+
+[role="screenshot"]
+image::images/indicators-table.png[Shows the Indicators table on the Intelligence page]
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+Threat intelligence is a research function that helps an organization make informed security decisions and actions. An organization's security operations center (SOC) team usually manages threat intelligence, which helps them understand current and emerging threats and recommend actions that will prevent, stop, and mitigate security threats.
```

Comment:
```suggestion
Threat intelligence is a research function that is responsible for understanding current and emerging threats, and recommending appropriate actions to strengthen the organization's security posture. Producing actionable threat Intelligence requires aggregating, analyzing, and synthesizing threat and vulnerability data from various sources and conducting proactive investigations.
```

Reply:
I feel like my second sentence can be better worded

Reply:
@dhru42 thanks for the additional content! Here's a potential revision:

```suggestion
Threat intelligence is a research function that is responsible for understanding current and emerging threats, and recommending appropriate actions to strengthen the organization's security posture. Threat intelligence must be actionable to be useful. Producing it requires gathering, analyzing, and synthesizing various threat and vulnerability data sources and conducting proactive investigations.
```